picasso-pollexor/build.gradle

```diff
@@ -31,14 +31,14 @@ android {
 }
 
 dependencies {
-  compile project(':picasso')
-  compile dep.pollexor
-  compile dep.supportAnnotations
-  testCompile dep.junit
-  testCompile dep.robolectric
-  testCompile dep.fest
-  testCompile dep.festAndroid
-  testCompile dep.pollexor
+  implementation project(':picasso')
+  implementation dep.pollexor
```

Comment:
Both of these should be `api` as types from both are present in the public API of this module.